CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# Adding `mobile-ui-testing-squad` as codeowners for UI tests related folders
+
```

Comment:
My two cents: I think we can do without this comment. The syntax is straightforward enough.

Reply:
Good point, I removed the comment in 298c320a726db48370035387fba5873587c6bece.